code/open_file.py

```diff
@@ -0,0 +1,80 @@
+# Actions for opening files in appropriate applications.
+import os
+import subprocess
+
+from talon import Context, Module, app
+
+# path to knausj root directory
+REPO_DIR = os.path.dirname(os.path.dirname(__file__))
+SETTINGS_DIR = os.path.join(REPO_DIR, "settings")
```

Comment:
Could wrap this in os.path.realpath(), so user can use a symlink for their settings directory.

Reply:
Not sure why this is necessary; it should support symlinks already? Also note that SETTINGS_DIR is derived in a different way in user_settings.py.

Reply:
Seems you're right, @nriley. I thought I remembered that Talon didn't follow symlinks by default, or maybe that it didn't support them if they pointed outside the user directory...something like that.

Reply:
Talon's watch function doesn't work with symlinks. Maybe that's what you're thinking off?